x/fees/types/msg.go

```diff
@@ -149,12 +163,19 @@ func (msg MsgUpdateDevFeeInfo) ValidateBasic() error {
 		return sdkerrors.Wrapf(err, "invalid contract address %s", msg.ContractAddress)
 	}
 
+	if ethermint.IsZeroAddress(msg.ContractAddress) {
```

Comment:
Correct me if I'm wrong, but there seems to be quite a bit of repetition on the address checks. Can we stick to DRY in this case and pull into a util func?

Reply:
How about adding in ethermint validation `ethermint.ValidateNonZeroAddress`?

Reply:
sounds good @loredanacirstea 👍

Reply:
Done in https://github.com/tharsis/ethermint/pull/1033, I will update the tests in evmos after the next release.